samples/client/petstore/typescript-angular-v11-provided-in-root/builds/with-npm/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@openapitools/typescript-angular-petstore",
+  "version": "1.0.0",
+  "description": "OpenAPI client for @openapitools/typescript-angular-petstore",
+  "author": "OpenAPI-Generator Contributors",
+  "keywords": [
+    "openapi-client",
+    "openapi-generator"
+  ],
+  "license": "Unlicense",
+  "scripts": {
+    "build": "ng-packagr -p ng-package.json"
+  },
+  "peerDependencies": {
+    "@angular/core": "^11.0.0",
+    "rxjs": "^6.6.0"
+  },
+  "devDependencies": {
+    "@angular/common": "^11.0.0",
+    "@angular/compiler": "^11.0.0",
+    "@angular/compiler-cli": "^11.0.0",
+    "@angular/core": "^11.0.0",
+    "@angular/platform-browser": "^11.0.0",
+    "ng-packagr": "^10.0.3",
```

Comment:
please re-generate the samples by running
```
mvn clean package
bin/generate-samples.sh
```

Reply:
done, forgot the run `mvn clean package` before :-/